### DISCUSSION TITLE
[FeatureRequest] Accelerometer + temp board

### DISCUSSION THREAD

**Discussion by HiAaronCody** [joeycastillo/Sensor-Watch#57] [Ideas]
Based on your updates on CrowdSupply, the accelerometer is top-of-the-list to be developed.

A small request: it would be awesome if the Accelerometer board also included the temp sensor.

I know you're not going to create a board for every combination of sensors, but these two are the most useful for me. Thanks!

**Comment by joeycastillo** [comment]
This would also be the most useful for me! It’s definitely my preference for an accelerometer board to also be able to sense temperature. 

One interesting note is that the LIS2DW I’m planning to use for the accelerometer board has an on-chip 12-bit temperature sensor. Alas, I haven’t been able to figure out how to turn the read values into useful temperature readings; the [data sheet](https://www.st.com/resource/en/datasheet/lis2dw.pdf) seems a bit sparse on that point). If we could figure that out, it could be a way to get good temperature data without the thermistor. This would leave a couple of pins available for UART test points, which I sense could be another valuable use of those pins.

**Reply by jhbruhn** [reply]
If I understand the datasheet correctly, the temperature is scaled linearly, where 0 is equal to 25°C (Toff does claim a very high variance here though), and 1 °C being represented with one bit (8-bit resolution mode), or 16 steps (12-bit resolution), according to TSDr. Does that make sense?

See here: https://github.com/STMicroelectronics/lis2dw12-pid/blob/master/lis2dw12_reg.c#L133

**Comment by newelldev** [comment]
Tossing it out here before it escapes my head: simple two-way communication with the watch via the accelerometer—tapping Morse code in and beeping Morse code in return?

**Comment by jacky4566** [comment]
Bumping this. @joeycastillo Where can i get a dev board? Would love to help with accelerometer integration. Can I ask why you are using I2C instead of SPI? Faster transfers means less wake time and you can ditch the pull up resistors for better power consumption.

Some ideas I have:
- Step counter, Pedometer. I have already written a pedometer algorithm for another application.
- Activity Reminders
- Record MAX G if you get hit by a bus
- Games?
